### ISSUE TITLE
Validation Errors and Warnings in Test

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#586]
PK: [4cb4ba22-c60e-4652-ac9f-b2e8b7a2306a](https://arax.ncats.io/beta/?r=4cb4ba22-c60e-4652-ac9f-b2e8b7a2306a)
UI:  https://ui.test.transltr.io/main/results?l=Metformin&i=PUBCHEM.COMPOUND:4091&t=4&q=4cb4ba22-c60e-4652-ac9f-b2e8b7a2306a

ARAGORN:
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/ec9cfad4-6a19-49d8-be8b-751aa3f477fb)

ARAX:
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/47ccb692-a58f-4ec9-a055-559a6c4c9370)

BTE:
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/dd4873da-2d1a-4d37-b84f-e6aa7f1bbdcd)

**Comment by saramsey** [comment]
Based on our team's investigation, one issue that is causing a validation error in the ARAX TRAPI result is that there are two result nodes, `UMLS:C1709820` ("ROS1 wt Allele") and `UMLS:C3890397` ("PRKAA1 wt Allele") both of which are annotated in the TRAPI result as having a `category` of `biolink:GenomicEntity` (which is not allowed because `GenomicEntity` is a "mixin" per the [Biolink YAML model](https://github.com/biolink/biolink-model/blob/85d7dab438ac54411550aa3a19e706553c55fc37/biolink-model.yaml#L6792). 

So, where is the "genomic entity" category coming from?  Based on our investigation, we do not think it is coming from the current production RTX-KG2 graph (i.e., RTX-KG2.8.4c).  But I notice that when I query the SRI Node Normalization service with this request:
```
https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes?curie=UMLS%3AC1709820&conflate=true&drug_chemical_conflate=false&description=false
```
I get back "Genomic Entity" in the response:
```
{
  "UMLS:C1709820": {
    "id": {
      "identifier": "UMLS:C1709820",
      "label": "ROS1 wt Allele"
    },
    "equivalent_identifiers": [
      {
        "identifier": "UMLS:C1709820",
        "label": "ROS1 wt Allele"
      }
    ],
    "type": [
      "biolink:GenomicEntity"
    ]
  }
}
```
The same goes for when I query with the other CURIE:
```
https://nodenormalization-sri.renci.org/1.4/get_normalized_nodes?curie=UMLS%3AC3890397&conflate=true&drug_chemical_conflate=false&description=false
```

I get back a response with "Genomic Entity":
```
{
  "UMLS:C3890397": {
    "id": {
      "identifier": "UMLS:C3890397",
      "label": "PRKAA1 wt Allele"
    },
    "equivalent_identifiers": [
      {
        "identifier": "UMLS:C3890397",
        "label": "PRKAA1 wt Allele"
      }
    ],
    "type": [
      "biolink:GenomicEntity"
    ]
  }
}
```
Tagging and assigning @gaurav and the SRI crew on the node normalization issue.

There are other validation errors, however; I need to investigate those errors as some of those may in fact come from one of our services; so I'll leave myself assigned on the issue for now.

**Comment by saramsey** [comment]
Since all signs point to SRI Node Normalizer service at this point (at least, as concerns the validation _error_), I'm deassigning myself for now. If something else comes up that needs our team's input, please LMK.

**Comment by gaurav** [comment]
Thanks so much for the detailed analysis, @saramsey! Yup, this is definitely a NodeNorm issue: the problem is that [UMLS semantic type T028 "Gene or Genome"](https://uts.nlm.nih.gov/uts/umls/semantic-network/T028) is [mapped to the mixin biolink:GenomicEntity](https://github.com/biolink/biolink-model/blob/ddc2dc78e89153286ce57fcaafe4fc7113665753/biolink-model.yaml#L6792-L6800) in the Biolink model. As long as a clique has a non-mixin semantic type as well, all will be well, but if a clique (like the ones in this ticket) only has T028 as a semantic type, we will give it a type of biolink:GenomicEntity and it will not have any parent types. I've opened a ticket in the Babel repository to figure out how widespread this problem is and whether simplify modifying the Biolink mapping will fix it, or if a more sophisticated fix is necessary (https://github.com/TranslatorSRI/Babel/issues/196).

**Comment by saramsey** [comment]
My $0.02: I don't think GenomicEntity should be a mixin. Can we just make it a concrete class?  It is useful.

**Comment by gaurav** [comment]
That's a good idea! I've opened that issue at https://github.com/biolink/biolink-model/issues/1405

**Comment by sstemann** [comment]
@sierra-moxon @gaurav do you guys think this will change for lobster1?

**Comment by sstemann** [comment]
what about Octopus2? @sierra-moxon @gaurav

**Comment by sierra-moxon** [comment]
This is a fairly difficult change to the model as it represents a break in the ChemicalEntity/BiologicalEntity hierarchy.  Moving to hammerhead.

**Comment by gaurav** [comment]
I don't think this made it into Hammerhead, so I think we should push this into the next phase.

**Comment by sstemann** [comment]
Hammerhead, Test 

https://ui.test.transltr.io/results?l=Metformin&i=CHEBI:6801&t=4&r=0&q=e0eeddf5-b9fa-427a-bc75-406d10f85071

![image](https://github.com/user-attachments/assets/1f2b19b9-a62d-485f-8862-9753e9a2c1ac)
